Thanks for adding this, but it still fails the detector tests we have:

```
[pr/1370●] » go test -tags detectors ./pkg/detectors/mockaroo
--- FAIL: TestMockaroo_FromChunk (0.99s)
    --- FAIL: TestMockaroo_FromChunk/found,_verified (0.45s)
        mockaroo_test.go:100: Mockaroo.FromData() found, verified diff: (-got +want)
             [
              {
               DetectorType: 753,
               DetectorName: "",
               DecoderType: 0,
            -  Verified: false,
            +  Verified: true,
               Raw: [
               ],
               RawV2: [
               ],
               Redacted: "",
               ExtraData: {
               },
               StructuredData: nil,
              },
             ]
FAIL
FAIL    github.com/trufflesecurity/trufflehog/v3/pkg/detectors/mockaroo 1.159s
FAIL
```

Changing the verification URL to `https://my.api.mockaroo.com` works though, from my testing earlier today.

--- a/pkg/detectors/mockaroo/mockaroo.go
+++ b/pkg/detectors/mockaroo/mockaroo.go
@@ -3,6 +3,7 @@ package mockaroo
 import (
 	"context"
 	"encoding/json"
+	"fmt"
 	"net/http"
 	"regexp"
 	"strings"
@@ -48,18 +49,17 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 		}
 
 		if verify {
-			req, err := http.NewRequestWithContext(ctx, "GET", "https://api.mockaroo.com/api/types", nil)
+			req, err := http.NewRequestWithContext(ctx, "GET", fmt.Sprintf("https://api.mockaroo.com/api/types?key=%s", resMatch), nil)
 			if err != nil {
 				continue
 			}
-			req.Header.Add("X-API-Key", resMatch)
 			res, err := client.Do(req)
 			if err == nil {
 				defer res.Body.Close()
 				if res.StatusCode >= 200 && res.StatusCode < 300 {
-					var t []typeResponse
+					var t typeRes
 					err = json.NewDecoder(res.Body).Decode(&t)
-					if err == nil && len(t) > 0 {
+					if err == nil && len(t.Types) > 0 {
 						s1.Verified = true
 					}
 				} else {
@@ -81,14 +81,10 @@ func (s Scanner) Type() detectorspb.DetectorType {
 	return detectorspb.DetectorType_Mockaroo
 }
 
-type typeResponse struct {
-	Name       string      `json:"name"`
-	Parameters []parameter `json:"parameters"`
-}
-
-type parameter struct {
-	Name        string      `json:"name"`
-	Type        string      `json:"type"`
-	Description string      `json:"description"`
-	Default     interface{} `json:"default"`
+type typeRes struct {
+	Types []struct {
+		Name       string        `json:"name"`
+		Type       interface{}   `json:"type"`
+		Parameters []interface{} `json:"parameters"`
+	} `json:"types"`
 }

